projects/proj-more-git.md

```diff
@@ -2,6 +2,10 @@
 
 # Project: More Git
 
-1. Drew's forked repo of the new Sparkbox site's content prototype to your personal account: <https://github.com/drewclemens/sparkbox-website>
-2. Add your own profile page to the site
-3. Submit a pull request for review
\ No newline at end of file
+1. Make a dotfiles directory and store any relevant dotfiles that you'll make changes to inside of that directory (e.g. .bash_profile, .bashrc, .gitconfig, .gitignore).
+2. Initialize a new repository inside of your dotfiles directory.
+3. Create symlinks for any dotfiles that you've moved from your root.
+4. Add customizations to your dotfiles as you see fit.
+5. Push dotfiles directory to GitHub.
```

Comment:
_directory_ should probably be _repo_ or _repository_.  Something like: "Push your `dotfiles` repository to Github."